docs/api/web-frame-main.md

```diff
@@ -231,6 +231,29 @@ A `string` representing the [visibility state](https://developer.mozilla.org/en-
 
 See also how the [Page Visibility API](browser-window.md#page-visibility) is affected by other Electron APIs.
 
+#### `frame.unresponsiveDocumentJSCallStack` _Readonly_ _Experimental_
```

Comment:
this is a strange API name for this shape. Does this only give you a call stack if the document is unresponsive? what happens if you access it while the document is _not_ unresponsive? If it's required that the document be unresponsive for this to work, should this instead be passed as part of the unresponsive event? How should a caller know whether it's available or not?